From: Marcel Holtmann <marcel@holtmann.org>
Date: Tue, 25 Mar 2014 20:21:23 -0700
Subject: hwdb: Update database of Bluetooth company identifiers

(cherry picked from commit f89b42ec0fd51feab5566d6bdbacee101023c037)
(cherry picked from commit cc9bdcf31e33b7012a29855c22a0a0c7ca49bcac)
---
 hwdb/20-bluetooth-vendor-product.hwdb | 18 ++++++++++++++++++
 1 file changed, 18 insertions(+)

diff --git a/hwdb/20-bluetooth-vendor-product.hwdb b/hwdb/20-bluetooth-vendor-product.hwdb
index 7d86f10..b928f5a 100644
--- a/hwdb/20-bluetooth-vendor-product.hwdb
+++ b/hwdb/20-bluetooth-vendor-product.hwdb
@@ -923,3 +923,21 @@ bluetooth:v0130*
 
 bluetooth:v0131*
  ID_VENDOR_FROM_DATABASE=Cypress Semiconductor Corporation
+
+bluetooth:v0132*
+ ID_VENDOR_FROM_DATABASE=MADS Inc
+
+bluetooth:v0133*
+ ID_VENDOR_FROM_DATABASE=Blue Maestro Limited
+
+bluetooth:v0134*
+ ID_VENDOR_FROM_DATABASE=Resolution Products, Inc.
+
+bluetooth:v0135*
+ ID_VENDOR_FROM_DATABASE=Airewear LLC
+
+bluetooth:v0136*
+ ID_VENDOR_FROM_DATABASE=ETC sp. z.o.o.
+
+bluetooth:v0137*
+ ID_VENDOR_FROM_DATABASE=Prestigio Plaza Ltd.
